c_emulator/riscv_sail.h

```diff
@@ -67,6 +67,7 @@ extern mach_bits zsepc, zstval;
 extern mach_bits zfloat_result, zfloat_fflags;
 
 struct zMcause {mach_bits zMcause_chunk_0;};
-struct zMcause zmcause, zscause;
```

Comment:
Why split?

Reply:
Multiple declarations on one line are generally considered bad style, so I thought I'd split it. Can of course change it back to what it was.